lib/datadog/ci/ext/environment.rb

```diff
@@ -293,9 +305,21 @@ def extract_bitrise(env)
             Datadog::Ext::Git::TAG_REPOSITORY_URL => env['GIT_REPOSITORY_URL'],
             Datadog::Ext::Git::TAG_COMMIT_SHA => commit,
             Datadog::Ext::Git::TAG_BRANCH => branch,
-            Datadog::Ext::Git::TAG_TAG => env['BITRISE_GIT_TAG']
+            Datadog::Ext::Git::TAG_TAG => env['BITRISE_GIT_TAG'],
+            Datadog::Ext::Git::TAG_COMMIT_MESSAGE => env['BITRISE_GIT_MESSAGE']
           }
         end
+
+        def branch_or_tag(branch_or_tag)
```

Comment:
Minor: Should this be `private`?

Reply:
I made it private, but now the `extract_*` methods can't call it.
I think this will be better addressed when
```ruby
EnvironmentInfo.new(
  tag_branch: ...,
  tag_commit_sha: ...,
  #...
)
```
is introduced.